helm/openwhisk/templates/invoker-role.yaml

```diff
@@ -34,6 +34,9 @@ rules:
 - apiGroups: [""]
   resources: ["pods"]
   verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+- apiGroups: [""]
+  resources: ["events"]
+  verbs: ["get", "list"]
```

Comment:
Seems our downstream doesn't have this changes,
So we changed the events k8s resource's permission manually?

Reply:
Our downstream use another invoker-role.yaml which already applied above configuration.